### DISCUSSION TITLE
棱镜万事屋开张啦！

### DISCUSSION THREAD

**Discussion by yutong903** [share121/inter-knot#1705] [其他]
各位绳匠们，你们好，想必你们一定在《绝区零》中做了不少委托吧
哎，我突然有个想法，
我决定在线上开一家万事屋，所有，从今天起，棱镜万事屋正式开张啦
各位有委托可以找我们哦
（目前只能在浙江龙游接委托，真是非常抱歉）
<img width="199" alt="t0128a9de8a40154f66" src="https://github.com/user-attachments/assets/f38409af-1f30-4c36-8371-0fff16089d2a">

**Comment by lllliufeng** [comment]
薄纱狡兔屋指日可待（

**Comment by dolokong** [comment]
不是，你开万事屋怎么还用友商的员工头像啊（恼）

**Comment by Li-Ming341** [comment]
不是，你来真的？

**Comment by nostalgiatan** [comment]
真的，还是？

**Comment by anything12121** [comment]
离谱了

**Comment by yutong903** [comment]
统一回复一下，是真的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/8b9838175cb6312df34c32648a8e6649_1050729359995605746.png" alt="景元 吸猫" width="75" >

**Reply by nostalgiatan** [reply]
什么业务范围呢？

**Reply by yutong903** [reply]
我能做的到的一般都接😋
